pkg/snapshot/v1alpha1/snapshot_template_engine.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// This file has snapshot specific implementation of cas template engine
+// Package snapshot has snapshot specific implementation of cas template engine
```

Comment:
This change is not required. Just add one empty line here.

Reply:
Done